app/src/main/res/values/strings.xml

```diff
@@ -558,6 +558,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <!-- Quests -->
     <!-- sorted alphabetically by title -->
 
+    <string name="quest_accepts_cards">Can you pay here with card?</string>
+    <string name="quest_accepts_cards_debit_and_credit">Credit and debit cards allowed</string>
```

Comment:
At least in American English, you’d say credit and debit cards are “accepted”, not “allowed”.

Reply:
thanks, English improved (I really, really appreciated feedback about language)